packages/datagateway-search/src/search/datePicker.component.tsx

```diff
@@ -112,94 +160,58 @@ export function SelectDates(props: DatePickerCombinedProps): JSX.Element {
       <LocalizationProvider dateAdapter={AdapterDateFns}>
         <>
           <DatePicker
-            maxDate={endDate || new Date()}
-            inputFormat="yyyy-MM-dd"
-            mask="____-__-__"
+            format="yyyy-MM-dd"
             value={startDate}
+            maxDate={endDate || new Date()}
             onChange={(date) => {
               handleChange(date as Date, 'startDate');
             }}
-            componentsProps={{
+            slots={{
+              actionBar: CustomClearButton,
+              textField: CustomTextField,
+            }}
+            slotProps={{
               actionBar: {
                 actions: ['clear'],
               },
-            }}
-            components={{
-              ActionBar: CustomClearButton,
-            }}
-            renderInput={(props) => {
-              const error =
-                // eslint-disable-next-line react/prop-types
-                (props.error || invalidDateRange) ?? undefined;
-              let helperText = t('searchBox.invalid_date_message');
-              if (invalidDateRange)
-                helperText = t('searchBox.invalid_date_range_message');
-
-              return (
-                <TextField
-                  {...props}
-                  inputProps={{
-                    // eslint-disable-next-line react/prop-types
-                    ...props.inputProps,
-                    placeholder: t('searchBox.start_date'),
-                    'aria-label': t('searchBox.start_date_arialabel'),
-                  }}
-                  onKeyDown={handleKeyDown}
-                  sx={sideLayout ? {} : { py: 1, width: '178px' }}
-                  variant="outlined"
-                  error={error}
-                  // eslint-disable-next-line react/prop-types
-                  {...(error && { helperText: helperText })}
-                />
-              );
+              textField: {
+                inputProps: {
+                  invalidDateRange,
+                  sideLayout,
+                  t: [t],
```

Comment:
We can just pass `t` here, no need to put it into an array again - this makes the destructuring comment I made easier too